database/migrations/2014_10_12_100000_create_password_resets_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePasswordResetsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using short method names like CreatePasswordResetsTable::up(). The configured minimum method name length is 3.](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)